src/Setup/Data/Tool.purs

```diff
@@ -153,8 +159,8 @@ installMethod tool version = do
   -- Example: "v0.13.2", "0.15.2"
   formatTag :: String
   formatTag = do
-    let versionStr = Version.showVersion version
-    if tool `elem` [ PureScript, Zephyr, Psa ] then
+    let versionStr = VersionField.showVersionField versionField
+    if tool `elem` [ PureScript, Zephyr ] then
```

Comment:
I _think_ the `Psa` case here was unreachable—`formatTag` is only referenced in the `PureScript`, `Spago` and `Zephyr` cases. If it is needed, I would have to expand this a bit so not to prepend `v` to `latest`.

Reply:
I think we should either:

1. Inline `formatTag` so that it's obviously only used for the `Tarball` installation method, because that's the only time we need to bother with the specific format of the tag, or
2. Keep `Psa` in here, and add `PursTidy`, because they do use a `v` prefix on their tags (unlike, for example, [how Spago formats tags](https://github.com/purescript/spago/releases/tag/0.20.3) with no `v` prefix).

My preference would be (1) because we really don't need the tag for the `NPM` install method.